apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -73,6 +74,7 @@ class SectionProgress extends Component {
               section={section}
               scriptData={scriptData}
             />
+            <SummaryViewLegend/>
```

Comment:
Next step is to use the current script to determine whether to pass `showCSFProgressBox` here.